pylti/common.py

```diff
@@ -216,6 +217,7 @@ def post_message(consumers, lti_key, url, body):
     :param body: xml body
     :return: success
     """
+    # pylint: disable=unused-variable
```

Comment:
You should actually fix the issues rather than just disabling everything. For example remove all the unused variables.